ckanext/external_storage/react/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
+    "axios": "^0.21.1",
+    "frictionless.js": "git+https://github.com/frictionlessdata/frictionless-js.git",
+    "giftless-client": "git+https://github.com/datopian/giftless-client-js.git",
```

Comment:
Are we ok linking to the latest version of `frictionless.js` and `giftless-client` or shall we pin them?